collect_app/src/main/java/org/odk/collect/android/activities/FirstLaunchActivity.kt

```diff
@@ -69,6 +77,19 @@ class FirstLaunchActivity : LocalizedActivity() {
                 }
             }
 
+            lifecycleScope.launch {
```

Comment:
We could use `LiveData` here instead by either converting (with `toLiveData()`) in the `FirstLaunchViewModel` or here. I think that generally ends up being simpler in the UI code.

Reply:
Done.